routes/panel.js

```diff
@@ -89,10 +90,12 @@ router.post('/signup', function(req, res) {
 
 /***** FAQ *****/
 
+/*
 router.get('/faq', function(req, res) {
 	var path = require('path');
 	res.sendFile(path.resolve('public/panel/faq.html'));
 });
+*/
```

Comment:
If we're no longer using code, let's delete it rather than commenting it out. If we need to go back, that's what git diffs are for!